tests/test_delb_plugin.py

```diff
@@ -0,0 +1,66 @@
+import pytest
+
+
+from delb import Document
+from snakesist import ExistClient
+
+
+@pytest.mark.usefixtures("db")
+def test_delete_document():
+    url = "existdb://admin:@localhost:8080/test_collection/test_document.xml"
+    document = Document(url)
+    document.existdb_delete()
+
+
+@pytest.mark.usefixtures("db")
+def test_load_document_from_url():
+    url = "existdb://admin:@localhost:8080/test_collection/test_document.xml"
+    document = Document(url)
+
+    assert document.config.source_url == url
```

Comment:
should this property just be bound to the `Document` instances, e.g. `document.source_url`? this here was modeled after the `https_loader`'s behaviour. the interfaces don't allow loaders to interact with the document instance though.

Reply:
Sorry for rushing.
 
I hope I'm not missing something here, but I don't see why `source_url` couldn't be bound to a `Document` instance. I'm not sure I understand the way the loaders work, I'll have to take a closer look at that.